spaces/domains.json

```
@@ -130,4 +130,5 @@
   "poll.santiment.net": "santiment.eth",
   "voting-bmxx.multiplier.finance": "mcl-multiplierfinance.eth",
   "vote.cndy.finance": "candyfinance.eth"
+  "riseprotocol.io/governance': "riseprotocol.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.cndy.finance": "candyfinance.eth",
  "riseprotocol.io/governance': "riseprotocol.eth"
```